lib/rubocop/cop/rspec/iterated_expectation.rb

```diff
@@ -28,6 +28,13 @@ class IteratedExpectation < Base
           )
         PATTERN
 
+        # @!method each_numblock?(node)
+        def_node_matcher :each_numblock?, <<-PATTERN
+          (numblock
+            (send ... :each) 1 $(...)
```

Comment:
hm, what does the 1 means in the pattern 🤔

Reply:
It is possible that my matcher is not written in the right way.
If the code is as follows:
```ruby
it 'validates users' do
  [user1, user2, user3].each { expect(_1).to be_valid }
end
```

It is expressed in AST as follows:
```
s(:numblock,
  s(:send,
    s(:array,
      s(:send, nil, :user1),
      s(:send, nil, :user2),
      s(:send, nil, :user3)), :each), 1, // 👈
  s(:send,
    s(:send, nil, :expect,
      s(:lvar, :_1)), :to,
    s(:send, nil, :be_valid)))
```

so adjusting for this `1` 🤔

Reply:
It is changed to `_`. What do you think?

Reply:
Hah, it seems `1` indicates the number of arguments (the highest numbered argument) used.
```
[user1, user2, user3].each { expect(_1).to be_valid(_2) }
```
produces `2` instead
```
[user1, user2, user3].each { expect(_1).to be_valid(_3) }
```
results in a 3.

if we care that just _1 is used, then the original version is fine. If we don't care, then it's `_`

Reply:
I see. I understand very well, thank you.
we are going to use `_` here.